runbooks/koji/upgrade.md

```diff
@@ -0,0 +1,83 @@
+Koji upgrade
+==============
+
+Issue
+-----
+* You want to upgrade [Koji](http://koji.katello.org) to a newer version
+
+Resolution
+----------
+* Shut down koji services on `koji-fedora28-builder` and `kojibuilder1`.
+  * `koji-fedora28-builder`
+    ```
+    # systemctl stop kojid
+    ```
+  * `kojibuilder1`
+	```
+    # systemctl stop kojid
+	# systemctl stop kojira
+	# systemctl stop httpd
+	# systemctl stop postgresql
+    ```
+* Take snapshots in aws.
+  * EBS -> Volumes
+  * Right click on “koji 4 root volume” -> create snapshot
+  * Right click on “koji4 data volume /mnt/koji” -> create snapshot
+    * You can still use the host os while snapshot is `pending`,
+		it refers to the transfer to amazon s3.
+* update vms to latest software.
+  * `kojibuilder1`
+	* `# yum update`
+	* Remove `/etc/httpd/conf.d/{kojihub,kojiweb,ssl}.conf`
```

Comment:
To avoid that issue, should we keep the files but keep a blank content (`# This file is intentionally blank`). That prevents yum from overwriting it if they used the `noreplace` part in the RPM spec.

Reply:
Ideally, in the future, we'd actually use these files instead.  The files in /etc/httpd/conf.modules.d/ are supposed to modify module config parameters, not site config parameters